### PR TITLE
Add gems to gemfile

### PR DESCRIPTION
**Pull request by PerezIgnacio**

### CODE REVIEW

File: config/rails_best_practices.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+AddModelVirtualAttributeCheck: { }
+AlwaysAddDbIndexCheck: { }
+#CheckSaveReturnValueCheck: { }
```
Comment (brunvez) [inline config/rails_best_practices.yml]: Remove commented code

File: lib/tasks/code_analysis.rake [hunk 2]
```diff
@@ -0,0 +1,6 @@
+task :code_analysis do
+  # sh 'bundle exec brakeman . -z -q'
```
Comment (brunvez) [inline lib/tasks/code_analysis.rake]: Same

**Comment by brunvez** [comment]
Commit messages should be written in an imperative mood. Try something like `Add gems to Gemfile`

File: .reek.yml [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+  UnusedParameters:
+    enabled: true
+    exclude: []
+  UnusedPrivateMethod:
+    enabled: false
+  UtilityFunction:
+    enabled: false
+
+exclude_paths:
+  - config
```
Comment (fedeagripa) [inline .reek.yml]: please remember to add empty line at end of file

File: .rubocop.yml [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  Enabled: false
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
+Style/ModuleFunction:
+  Enabled: false
+
+Style/ReturnNil:
+  Enabled: true
```
Comment (fedeagripa) [inline .rubocop.yml]: same here

File: Gemfile [hunk 5]
```diff
@@ -48,6 +55,11 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
+  gem 'rspec-rails', '~> 3.8', '>= 3.8.2'
```
Comment (jpascual1994) [inline Gemfile]: I think you need this gem in the test environment too

**Pull request merged**